src/VisualStudio/IntegrationTest/TestUtilities/InProcess/VisualStudioWorkspace_InProc.cs

```diff
@@ -103,7 +103,7 @@ public void CleanUpWorkspace()
             {
                 LoadRoslynPackage();
 
-                var hook = _visualStudioWorkspace.Services.GetRequiredService<IWorkpacePartialSolutionsTestHook>();
+                var hook = _visualStudioWorkspace.Services.GetRequiredService<IWorkspacePartialSolutionsTestHook>();
```

Comment:
@sharwell said this is not used, right? Then let's delete it here, or at least add some comments to explain what's going on.

Reply:
would prefer to make that a separate PR :)